test/test_core.py

```diff
@@ -7774,6 +7774,33 @@ def test_embind_val_assignment(self):
     err = self.expect_fail([EMCC, test_file('embind/test_val_assignment.cpp'), '-lembind', '-c'])
     self.assertContained('candidate function not viable: expects an lvalue for object argument', err)
 
+  def test_embind_val_cross_thread(self):
+    self.emcc_args += ['--bind', '-pthread']
+    self.setup_node_pthreads()
```

Comment:
I think we have a `@node_pthreads` decorator for this

Reply:
Ah right.